olive/workflows/run/run.py

```diff
@@ -165,8 +165,10 @@ def run_engine(config: RunConfig, data_root: str = None):
 
         AzureMLSystem.olive_config = config.to_json()
 
-    no_evaluation = engine.evaluator_config is None and all(
-        pass_config.evaluator is None for pass_config in config.passes.values()
+    no_evaluation = (
+        engine.evaluator_config is None
+        and config.passes
```

Comment:
when the pass is None?

Reply:
If there no passes given in input model config, we need to check `config.passes` since we need to enable auto opt. See the PR description item 1.

Reply:
but the no_evaluation only used by create_accelerators? how it related to auto opt?

Reply:
no_evaluation is not related with auto opt. I just triggered the issue when I enable auto opt.

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/8749bd20-1254-40a5-bc13-a3ff50df742d)
That is introduced in https://github.com/microsoft/Olive/pull/937